sklearnex/preview/linear_model/tests/test_ridge.py

```diff
@@ -0,0 +1,54 @@
+# ===============================================================================
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+import numpy
+import pytest
+
+from onedal.tests.utils._dataframes_support import (
+    _convert_to_dataframe,
+    get_dataframes_and_queues,
+)
+
+
+@pytest.mark.parametrize("alpha", [0.5, 1.0, 5.0])
+@pytest.mark.parametrize("dataframe,queue", get_dataframes_and_queues())
+def test_ridge_sklearnex_sklearn_conformance(alpha, dataframe, queue):
```

Comment:
We already doing this via sklearn testing

Reply:
Agreed, we don't need to define our own testing to validate on scikit-learn. Can get creative with devising tests here. I recommend basing off of existing linear regression coverage: https://github.com/intel/scikit-learn-intelex/blob/main/sklearnex/linear_model/tests/test_linear.py. With linear regression there is simple solution, ridge is a bit more complicated but can be done using numpy linalg operations, could be a good place to start